### DISCUSSION TITLE
GitHub Student Developer Pack cannot access Copilot.

### DISCUSSION THREAD

**Discussion by doveppp** [community/community#113627] [GitHub Education] [answered]
### Select Topic Area

Question

### Body

My student benefits have been approved, but I still can't use Copilot. Clicking on "start free trial" also doesn't do anything.
![image](https://github.com/community/community/assets/54977106/2aba1321-b317-4f4b-a625-14a364e3e211)
![image](https://github.com/community/community/assets/54977106/007cc973-8958-47a9-a54b-33500b4a677d)

**Comment by ZuhairTarif** [comment]
Use it as an extension and activate it from there. Install the Copilot extension in vsCode, for example, and then log in using GitHub. After that, you can see that it will activate automatically.

**Reply by doveppp** [reply]
vsCode show:
2024-03-22 11:56:51.616 [info] [auth] Invalid copilot token: missing token: 403 
2024-03-22 11:56:51.707 [error] [default] Extension activation failed: "Contact Support. You are currently logged in as doveppp."

**Reply by lx32056127** [reply]
My Github Copilot does the same thing, and I'm still a paid personal user!

Has this annoying problem been solved?

**Comment by doveppp** [comment, marked as answer]
Resolved.

Just contact GitHub Support.
